### ISSUE TITLE
[Feature Request] [Mod Compatibility] pop-up UI overlapping hearts

### ISSUE THREAD

**Issue by NotoriusQ** [ichttt/FirstAid#116]
This one might be kinda far out, but the numbers get hard for me to read.

If the hearts in the UI had a config option to overlay instead of changing to numbers at high values, they could use the Mantle or HealthOverlay colors for overlapping hearts, both absorption and health.
The missing hearts could still be 'absent' (dark), so you know what your max health is when only missing a little bit, but your hearts on a given limb would change color based on which tier 'health bar' you were on. 

Or however you could make it work!
Thanks for an awesome mod :D

**Comment by ichttt** [comment]
Yeah that's a nice idea, but it sounds like a lot of work, as the render code currently is designed for 2 rows at most.
It's a bit low prio since it does at least work in the current state and it only affects a small % of people (people with very much health), AND I would have to rewrite the health renderer...
